notebooks/Welcome Julia - Intro to Gadfly.ipynb

```diff
@@ -5217,13 +5217,15 @@
  ],
  "metadata": {
   "kernelspec": {
-   "codemirror_mode": "julia",
-   "display_name": "Julia",
+   "display_name": "Julia 0.4.0-dev+3395",
    "language": "julia",
    "name": "julia"
   },
-  "signature": "sha256:c57641129fc82564af9fa6a1c77c6097104dc130acb0a73081e18b0aa9ac51c7"
```

Comment:
Not, old format, but trusting is now in sqlite. 
We should run a `ipython notebook trust` or whatever.